src/reviewers/review-utils.ts

```diff
@@ -19,3 +30,65 @@ export function calculateOverallConclusionCode(codes: ConclusionCode[]): Conclus
   }
   return ConclusionCode.UNKNOWN;
 }
+
+export function generateAlignmentPairs(config: Configuration, fisher: Fisher): FHITAlignmentPair[] {
+  return config.alignments
+    .map((alignment: FHITAlignment) => {
+      const [sender, receiver] = Object.entries(alignment);
+      const [senderAlias, senderId] = sender;
+      const [receiverAlias, receiverId] = receiver;
+
+      const senderSD = fisher.fishForFHIR(senderId);
+      const receiverSD = fisher.fishForFHIR(receiverId);
+
+      if (!(senderSD && receiverSD)) {
+        if (senderId === 'NONE' || receiverId === 'NONE') {
```

Comment:
This would miss the case where someone uses a bad ID and aligns it with 'NONE'. I think you might need to check:
```suggestion
        if (senderId === 'NONE' && receiverSD || receiverId === 'NONE' && senderSD) {
```